frontend/www/js/omegaup/api.js

```diff
@@ -227,13 +227,19 @@ export default {
 
     update: _call('/api/contest/update/'),
 
+    updateEndTimeForIdentity: _call('/api/contest/updateEndTimeForIdentity/'),
+
     users: _call('/api/contest/users/',
                  function(result) {
                    for (const user of result.users) {
-                     user.access_time =
-                         omegaup.OmegaUp.remoteTime(user.access_time * 1000);
+                     user.access_time = user.access_time == null ?
```

Comment:
qué tal mejor

```suggestion
                     if (user.access_time !== null) {
                       user.access_time =
                           omegaup.OmegaUp.remoteTime(user.access_time * 1000);
                     }
```

igual abajo.

Reply:
Listo